meteor-app/imports/ui/admin/app-selection.js

```diff
@@ -1,45 +1,123 @@
-import React from 'react';
-import { Component } from 'react';
-import { Button, Container } from 'semantic-ui-react'
-const debug = require('debug')('b2b:admin')
+import React from "react";
+import { Component } from "react";
+import { Button, Container } from "semantic-ui-react";
+import UploadXL from "/imports/ui/ordering/uploadXL";
+import { Meteor } from "meteor/meteor";
```

Comment:
You can't call Meteor methods from a component that renders, because the storybook entry will fail. Pass in a function from the container, and call that